cmake/config.cmake

```diff
@@ -1,47 +0,0 @@
-#--------------------------------------------------------------------
-#  Template custom cmake configuration for compiling
-#
-#  This file is used to override the build options in build.
-#  If you want to change the configuration, please use the following
-#  steps. Assume you are on the root directory. First copy the this
-#  file so that any local changes will be ignored by git
-#
-#  $ mkdir build
-#  $ cp cmake/config.cmake build
-#
-#  Next modify the according entries, and then compile by
-#
-#  $ cd build
-#  $ cmake ..
-#
-#  Then buld in parallel with 8 threads
-#
-#  $ make -j8
-#--------------------------------------------------------------------
-
-#---------------------------------------------
-# Backend runtimes.
-#---------------------------------------------
-
-# Whether enable CUDA during compile,
-#
-# Possible values:
-# - ON: enable CUDA with cmake's auto search
-# - OFF: disable CUDA
-# - /path/to/cuda: use specific path to cuda toolkit
-set(USE_CUDA OFF)
-
-#---------------------------------------------
-# Misc.
-#---------------------------------------------
-# Whether to build cpp unittest executables.
-set(BUILD_CPP_TEST OFF)
-
-# Whether to enable OpenMP.
-set(USE_OPENMP ON)
-
-# Whether to build PyTorch plugins.
-set(BUILD_TORCH ON)
```

Comment:
in `CMakeLists.txt`, it's `dgl_option(BUILD_TORCH "Build the PyTorch plugin" OFF)`. So before this change, `BUILD_TORCH` is always on?

Reply:
No, this cmake/config.cmake was never included.

Reply:
The included config.cmake were DGL_HOME/config.cmake and DGL_HOME/build/config.cmake.